changelog.txt

```diff
@@ -94,7 +102,7 @@ Changed
  - Project names have a default in anticipation of removing names entirely. Project names will be removed in signac 2.0 (#644).
  - ``Project.workspace`` is now a property, not a method (#685).
  - Continuous integration uses GitHub Actions instead of CircleCI (#776, #788).
- - Raise errors in testing when ``DeprecatedWarning``s or ``FutureWarning``s are raised (#713).
+ - Raise errors in testing when ``DeprecatedWarning`` or ``FutureWarning`` are raised (#713).
```

Comment:
```suggestion
 - Raise errors in testing when ``DeprecatedWarning`` or ``FutureWarning`` is raised (#713).
```
Although, in general, I wouldn't get in the habit of altering previous changelog entries ...

Reply:
The only reason I edited this was that it was raising a warning in our documentation builds. The backticks followed by an “s” were confusing Sphinx.